src/common/stripDiacritics.ts

```diff
@@ -6,5 +6,5 @@ export function stripDiacritics(text: string): string {
   return text
     .normalize('NFD')
     .replace(/[\u0300-\u036f]/g, '')
-    .replace(/[^a-zA-Z0-9\s]/g, '');
+    .replace(/[^a-zA-Z0-9\s-]/g, '');
```

Comment:
It turned out that this function stripped `-` dashes, too, which was inconvenient for some regular expression checks.

Reply:
Verified that there are no side effects on numerals and pronouns where this function is used.